src/serverenvironment.cpp

```diff
@@ -873,10 +907,6 @@ void ServerEnvironment::activateBlock(MapBlock *block, u32 additional_dtime)
 					elapsed_timer.position));
 		}
 	}
-
-	/* Handle ActiveBlockModifiers */
```

Comment:
Since blocks are more frequently activated only run the ABMs at their step intervals. I think this is better anyway.